### PR TITLE
update step function

### LINKED ISSUE #4716: Updating docs for Step function

### PR DESCRIPTION
**Pull request by PUNITBATRA**
Fixes #4716

### CODE REVIEW

**Comment by PUNITBATRA** [comment]
@domoritz I have made the suggested changes by you.can you review this?

File: src/compositemark/errorband.ts [hunk 1]
```diff
@@ -57,9 +57,9 @@ export interface ErrorBandConfig extends ErrorBandPartsMixins {
    * The line interpolation method for the error band. One of the following:
    * - `"linear"`: piecewise linear segments, as in a polyline.
    * - `"linear-closed"`: close the linear segments to form a polygon.
-   * - `"step"`: alternate between horizontal and vertical segments, as in a step function.
-   * - `"step-before"`: alternate between vertical and horizontal segments, as in a step function.
-   * - `"step-after"`: alternate between horizontal and vertical segments, as in a step function.
+   * - `"step"`: produces a piecewise constant function (a step function) consisting of alternating horizontal and vertical lines. The y-value changes at the midpoint of each pair of adjacent x-values.
+   * - `"step-before"`: produces a piecewise constant function (a step function) consisting of alternating horizontal and vertical lines. The y-value changes before the x-value.
+   * - `"step-after"`: produces a piecewise constant function (a step function) consisting of alternating horizontal and vertical lines. The y-value changes after the x-value.
```
Comment (domoritz) [inline src/compositemark/errorband.ts]: Now we have sentences that start with a lowercase letter and are still inconsistent (only the new sentences start with "produces".

Comment (PUNITBATRA) [reply src/compositemark/errorband.ts]: means I have to write "produces" as "Produces"?

Comment (domoritz) [reply src/compositemark/errorband.ts]: Yes. Otherwise it's awkward.

Comment (PUNITBATRA) [reply src/compositemark/errorband.ts]: Thanks, @domoritz now you can check.

**Comment by domoritz** [comment]
Thanks. @kanitw can you review and merge.

File: src/compositemark/errorband.ts [hunk 2]
```diff
@@ -57,9 +57,9 @@ export interface ErrorBandConfig extends ErrorBandPartsMixins {
    * The line interpolation method for the error band. One of the following:
    * - `"linear"`: piecewise linear segments, as in a polyline.
    * - `"linear-closed"`: close the linear segments to form a polygon.
-   * - `"step"`: alternate between horizontal and vertical segments, as in a step function.
-   * - `"step-before"`: alternate between vertical and horizontal segments, as in a step function.
-   * - `"step-after"`: alternate between horizontal and vertical segments, as in a step function.
+   * - `"step"`: Produces a piecewise constant function (a step function) consisting of alternating horizontal and vertical lines. The y-value changes at the midpoint of each pair of adjacent x-values.
```
Comment (kanitw) [inline src/compositemark/errorband.ts]: small p? (word after colon should not be capitalized.  -- same for other lines.

Comment (domoritz) [reply src/compositemark/errorband.ts]: It's odd to say "produces" for only a few but not all. We should remove it.

Comment (PUNITBATRA) [reply src/compositemark/errorband.ts]: @kanitw Now you can review this. I have removed "produces" as suggested.

**Comment by domoritz** [comment]
Looks good now

**Comment by domoritz** [comment]
@PUNITBATRA Can you merge master so we retrigger a build?

**Comment by PUNITBATRA** [comment]
@domoritz I rebase my pr to latest merge. you can merge this now :)

**Comment by domoritz** [comment]
Hmm, somehow tests are still failing.

**Comment by PUNITBATRA** [comment]
@domoritz you can see the details of travis log it is not related to this pr. what should I do now? It is something related to debug.ts.

**Comment by domoritz** [comment]
Hmm, master is passing, though.

**Comment by domoritz** [comment]
Redone in https://github.com/vega/vega-lite/pull/4761

**Pull request closed without merging**